api/company-user/test.js

```diff
@@ -160,4 +160,65 @@ describe('Acceptance :: Route :: Company User Utilities', () => {
       });
     });
   });
+
+  describe('POST /api/v1/company-user/change-password', () => {
+    let baseUrl = '/api/v1/company-user/change-password/';
+
+    it('should work when a valid request is made', async () => {
+      let passwords = {
+        newPassword:     'Test',
+        currentPassword: '1234'
+      };
+      let user = await (new CompanyUser({
+        email: 'test@aehr.org',
+        name:  {
+          first: 'Bob',
+          last:  'Ross'
+        },
+        password: passwords.currentPassword
+      })).save();
+
+      let res = await chai.request(app)
+      .post(`${baseUrl}`)
+      .set('X-Test-User', user._id.toString())
+      .send({
+        passwords
+      });
+
+      expect(res).to.have.status(204);
+
+      let updatedUser = await CompanyUser.findById(user._id);
+
+      const newPasswordChanged = await updatedUser.compareHash(passwords.newPassword);
+
+      expect(newPasswordChanged).to.equal(true);
+    });
+
+    it('should return a 400', async () => {
```

Comment:
```suggestion
    it('should return a 400 when a current password is invalid', async () => {
```
Please be more descriptive if it seems vague.